Add hbase-operator-tools to the HBase image

siegfriedweber opened the pull request:
# Description

Add hbase-operator-tools to the HBase image

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
- [x] Changes are OpenShift compatible
- [x] All added packages (via microdnf or otherwise) have a comment on why they are added
- [x] Things not downloaded from Red Hat repositories should be mirrored in the Stackable repository and downloaded from there
- [x] All packages should have (if available) signatures/hashes verified
- [ ] Does your change affect an SBOM? Make sure to update all SBOMs
```

lfrancke commented:
I'm building it now, it looks good in principle but has one problem that will be hard to overcome: HBase Operator Tools builds against Hadoop 2 I believe but to fix that would be rather annoying.

If it works (did you test that @siegfriedweber?) I'm fine with it for now as we'll get a fix for this automatically when we upgrade to HBase 2.5++

siegfriedweber commented:
I tested HBCK2 with `scheduleRecoveries` and `recoverUnknown`:

```shell
# Schedule ServerCrashProcedure (SCP) for the given RegionServers
/stackable/hbase/bin/hbase hbck \
  -j /stackable/hbase-operator-tools/hbase-hbck2/hbase-hbck2-$version.jar \
  scheduleRecoveries $regionservers

# Schedule ServerCrashProcedure (SCP) for RegionServers that are reported as unknown
/stackable/hbase/bin/hbase hbck \
  -j /stackable/hbase-operator-tools/hbase-hbck2/hbase-hbck2-$version.jar \
  recoverUnknown
```

lfrancke reviewed (commented):
Three things:

- One change request: Add the license to hbase/licenses/
- A suggestion but I'm fine merging it without: Add a shell script wrapper `hbck2` that does the necessary call to `hbase` with all the stuff wired up and put that shell in the PATH
- A suggestion: Should we add a note in the docs about this tool being included?

siegfriedweber commented:
> * One change request: Add the license to hbase/licenses/

added in 15de8ea507765e724d8ef3fffd779418836140d4

> * A suggestion but I'm fine merging it without: Add a shell script wrapper `hbck2` that does the necessary call to `hbase` with all the stuff wired up and put that shell in the PATH

added in 012da897daf7a24a4e06a3ac7c2306037c16a81f

> * A suggestion: Should we add a note in the docs about this tool being included?

added in stackabletech/hbase-operator#416

lfrancke commented on hbase/Dockerfile:
```diff
@@ -13,6 +15,7 @@ SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 # zip required for the log4shell stuff
 RUN microdnf update && \
     microdnf install \
+    gettext \
```
I thought I commented on this before but must have forgotten to submit.
Can you add a comment what this is needed for please?

siegfriedweber replied:
commented in 39a6a9e20136ec89cb169d7cff5dedc050ba50a0

lfrancke replied:
Thank you! Even better than what I did

lfrancke reviewed (changes requested):
Minor thing, otherwise looks good. I built and tested it, thank you!
